woocommerce/woocommerce
# Update WC Tasks regarding wccom connection

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

This PR adds these following changes:
- Add "Extend your store" task
- Remove the the [current inbox message](https://prnt.sc/qXKb-0dm6KHz).

<img width="714" alt="image" src="https://github.com/woocommerce/woocommerce/assets/1663428/63941867-6cff-42e1-bf69-001c435df83b">





### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Start fresh instance of wp-env: `pnpm -- wp-env clean all` (why? to ensure that existing Inbox state is empty)
2. Start wp-env: `pnpm -- wp-env start`
3. Verify that the changes are valid



### Changelog entry







<details>

#### Significance




#### Type




#### Message 


</details>

<details>

#### Comment 


</details>
